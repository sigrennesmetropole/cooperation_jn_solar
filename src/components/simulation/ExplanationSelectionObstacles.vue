<script setup lang="ts">
import SelectObstacles from '@/assets/illustrations/select-obstacles.jpg'
import deleted from '@/assets/icons/interface-delete.svg'
</script>

<template>
  <div>
    <img class="w-full" :src="SelectObstacles" alt="" />
    <div class="mt-6">
      <div class="flex flex-row items-center">
        <div
          class="flex flex-row justify-center items-center bg-rose-200 rounded-full w-2.5 h-2.5 mr-2"
        >
          <img class="w-1 h-1" :src="deleted" alt="" />
        </div>
        <p class="font-dm-sans text-sm font-normal">
          Zones ne pouvant pas accueillir de panneaux solaires.
        </p>
      </div>
    </div>
  </div>
</template>
