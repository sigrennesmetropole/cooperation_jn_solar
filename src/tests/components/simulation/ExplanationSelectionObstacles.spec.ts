import ExplanationSelectionObstacles from '@/components/simulation/ExplanationSelectionObstacles.vue'
import { describe, it, expect, beforeEach } from 'vitest'
import { mount, VueWrapper } from '@vue/test-utils'

describe('ExplanationSelectionObstacles.vue', () => {
  let wrapper: VueWrapper

  beforeEach(async () => {
    wrapper = mount(ExplanationSelectionObstacles)
  })

  it('renders the component correctly', () => {
    expect(wrapper.exists()).toBe(true)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
