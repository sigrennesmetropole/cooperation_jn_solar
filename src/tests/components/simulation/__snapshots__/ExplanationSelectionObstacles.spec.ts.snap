// Vitest Snapshot v1

exports[`ExplanationSelectionObstacles.vue > renders the component correctly 1`] = `
"<div><img class=\\"w-full\\" src=\\"/src/assets/illustrations/select-obstacles.jpg\\" alt=\\"\\">
  <div class=\\"mt-6\\">
    <div class=\\"flex flex-row items-center\\">
      <div class=\\"flex flex-row justify-center items-center bg-rose-200 rounded-full w-2.5 h-2.5 mr-2\\"><img class=\\"w-1 h-1\\" src=\\"/src/assets/icons/interface-delete.svg\\" alt=\\"\\"></div>
      <p class=\\"font-dm-sans text-sm font-normal\\"> Zones ne pouvant pas accueillir de panneaux solaires. </p>
    </div>
  </div>
</div>"
`;
